test/keys/ephemeral-keys.spec.js

```diff
@@ -60,4 +60,13 @@ describe('generateEphemeralKeyPair', () => {
       expect(secrets[0]).to.have.length(32)
     })
   })
+
+  it(`handles bad curve name`, async () => {
+    try {
+      await crypto.keys.generateEphemeralKeyPair('bad name')
+    } catch (err) {
+      return
```

Comment:
Should probably assert on the error code here to avoid [Pokemon Exception Handling](https://blog.codinghorror.com/new-programming-jargon/)..

Reply:
Sorry, should have added this comment yesterday.  Everything else LGTM

Reply:
Lucky you did - uncovered a bug, thanks!

Reply:
Phew!